pages/portfolio.html

```diff
@@ -1,12 +1,66 @@
+<!-- must include information about several projects 
+  you have completed at Ada (or elsewhere) with links 
+  to the GitHub repo (if available), descriptions, 
+  images, etc. -->
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>my portfolio</title>
+  <!-- <link href="/style/style.css" rel="stylesheet">  -->
+  <link href="/style/about.css" rel="stylesheet"> 
 </head>
 <body>
-  
+  <header>
+    <h1>portfolio</h1>
+        <nav>
+          <ul>
+            <li><a href="/pages/index.html">home</a></li>
+            <li><a href="/pages/portfolio.html">portfolio</a></li>
+            <li><a href="/pages/about.html">about</a></li>
+          </ul>
+        </nav>
+  </header>
+  <main>
+    <section>
+      <div class='container'>
+        <a href="https://github.com/jaekerrclayton/swap-meet"><img src="/images/ezgif-4-cbe5863d24.gif" alt='swap-meet-photo'/></a>
```

Comment:
From a usability standpoint, only linking the image is a little surprising. My inclination is to click the text, but the text isn't linked, and when positioned over the image, blocks the ability to click through to the image.

One thing we can do to check that thinks are laid out for usability is the temporary comment out our stylesheet link to see how the page renders without the rules, and make sure that the page is still usable.